Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -509,7 +509,12 @@ typedef NS_OPTIONS(NSUInteger, WKDisplayCaptureSurfaces) {
 @property (nonatomic, readonly) _UIFindInteraction *_findInteraction WK_API_AVAILABLE(ios(WK_IOS_TBA));
 @property (nonatomic, readwrite, setter=_setFindInteractionEnabled:) BOOL _findInteractionEnabled WK_API_AVAILABLE(ios(WK_IOS_TBA));
 
+@property (nonatomic, readonly) CALayer *_findOverlayLayer WK_API_AVAILABLE(ios(WK_IOS_TBA));
```

Comment:
Ambiguous name: "find the overlay layer" or "layer for find overlay". _layerForFindOverlay: would avoid the ambiguity.

Reply:
Renamed.